source/blog/2017-03-30-minishift-intro.html.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Introducing Minishift - Run OpenShift locally
+date: 2017-03-30 05:45 UTC
+author: budhram
+published: true
+comments: true
+tags: minishift, atomic-developer-bundle, openshift-local
+---
+
+[Atomic Developer Bundle (ADB)](https://github.com/projectatomic/adb-atomic-developer-bundle) has been a great tool so far in providing a prepackaged development environment filled with production-grade, pre-configured tools, that makes the container developer's lives easier.
+
+We would now like to introduce you to [Minishift](https://github.com/minishift/minishift) which provides a far better user experience as compared to ADB. We moved from ADB to Minishift, to improve the user experience as well as to address Vagrant based issues being faced by our users.
+
+Minishift is a CLI tool that helps you run OpenShift locally by running a single-node cluster inside a VM. You can try out OpenShift or develop with it, day-to-day, on your local host.
+
+It is actually a single binary which uses [libmachine](https://github.com/docker/machine/tree/master/libmachine) to provision a VM with a local, single node, [OpenShift Origin](https://github.com/openshift/origin) cluster. It uses [OpenShift local cluster management](https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md) underneath to bring up single node cluster.
+
+It's a platform agnostic CLI tool that supports multiple hypervisors like KVM for Linux, Xhyve and VMWare Fusion for Mac OS, HyperV for Windows and VirtualBox in all OS platforms. Detailed instructions about it can be found in the [Minishift documentation](https://github.com/Minishift/minishift#prerequisites).
```

Comment:
I have taken that from our README @LalatenduMohanty , let's remove from that too.